index.html

```diff
@@ -0,0 +1,201 @@
+                </p>
+                <h3 class="mt-5">2022.12.22(THU) <span>&#126;</span> 2022.12.30(FRI)</h3>
+                <p id="venue">&#64; Serena Hotel, Kampala Uganda</p>
+              </div>
+              <div class="col-1"></div>
+            </div>
+          </div>
+        </section>
+        <section class="program" id="program">
+          <div class="row program-title container-fluid mx-auto">
+            <div class="col"></div>
+            <div class="col col- mt-3 pt-3"><h3>Main Program</h3></div>
+            <div class="col"></div>
+          </div>
+          <div class="row program-title container mx-auto">
+            <div class="col"></div>
+            <div class="col"><hr id="line"></div>
+            <div class="col"></div>
+          </div>
+          <div class="container-fluid">
+            <div class="row mt-3 cards mx-auto programs">
+              <div class=" col-md-auto card">
+                <img src="./images/speak.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Lecture</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+              <div class="col-md-auto card">
+                <img src="./images/exhibit.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Exhibition</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+              <div class="col-md-auto card">
+                <img src="./images/forum.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Forum</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+              <div class="col-md-auto card">
+                <img src="./images/wkshop.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Workshop</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+              <div class="col-md-auto card">
+                <img src="./images/ignite.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Resources</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
```

Comment:
- [ ] As the requirements suggest, make sure you follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
  - Layout: composition and space between elements.
  
Figma design 
![image](https://user-images.githubusercontent.com/52143815/192085700-5883346f-c72d-44b2-987c-3b9525e400cc.png)

Your design
![FireShot Capture 002 - East Africa Conference - raw githack com](https://user-images.githubusercontent.com/52143815/192086114-dd470665-e91f-4a01-b3f8-6959d06a2482.png)